extended/src/main/java/apoc/load/util/XmlUtil.java

```diff
@@ -0,0 +1,514 @@
+package apoc.load.util;
+
+import apoc.export.util.BatchTransaction;
+import apoc.export.util.ExportConfig;
+import apoc.export.util.Reporter;
+import apoc.util.JsonUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.neo4j.graphdb.Entity;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.Relationship;
+import org.neo4j.graphdb.RelationshipType;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.procedure.TerminationGuard;
+
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamConstants;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.events.Attribute;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+import java.io.Reader;
+import java.lang.reflect.Array;
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+
+public class XmlUtil {
```

Comment:
Rinominala in XmlImportUtil, 
e fai un'altra classe XmlLoadUtil in cui ci metti il metodo `xmlXpathToMapResult` e tutte le cose che vengono richiamate da essa.

E ci metti come commento
```
    /**
     * Taken from <a href="https://github.com/neo4j/apoc/blob/dev/core/src/main/java/apoc/load/Xml.java">Xml</a>
     * placed in APOC Core
     */
```

sempre per il discorso che se ci sono dei bug / codice brutto, possiamo dire "we ciccio, l'ho preso da core, non è colpa mia"